website/templates/project/registrations.mako

```diff
@@ -81,27 +80,41 @@
                 <p>Started: <span data-bind="text: initiated"></span></p>
                 <p>Last updated: <span data-bind="text: updated"></span></p>
                 <span data-bind="if: requiresApproval">
-                    <div data-bind="if: isApproved">
-                        <div class="draft-status-badge bg-success"> Approved</div>
-                    </div>
-                    <div data-bind="ifnot: isApproved">
-                        <div class="draft-status-badge bg-warning"> Pending Approval </div>
-                    </div>
-                    <div data-bind="if: isPendingReview">
+                    <div data-bind="if: isPendingApproval">
                         <div class="draft-status-badge bg-warning"> Pending Review</div>
                     </div>
+                    <div data-bind="if: userHasUnseenComment">
+                        <div class="draft-status-badge bg-warning"> Unseen Comments</div>
+                    </div>
                 </span>
                 </small>
                 <div class="row">
                   <div class="col-md-10">
                     <a class="btn btn-info"
-                       data-bind="click: $root.editDraft"><i style="margin-right: 5px;" class="fa fa-pencil"></i>Edit</a>
+                       data-bind="visible: !isPendingApproval,
+                                  click: $root.editDraft">
+                      <i style="margin-right: 5px;" class="fa fa-pencil"></i>Edit
+                    </a>
+                    <a class="btn btn-info"
+                       data-bind="visible: isPendingApproval,
+                                  click: $root.previewDraft">
+                      <i style="margin-right: 5px;" class="fa fa-pencil"></i>Preview
+                    </a>
                     <button class="btn btn-danger"
-                            data-bind="click: $root.deleteDraft"><i style="margin-right: 5px;" class="fa fa-times"></i>Delete</button>
+                            data-bind="click: $root.deleteDraft">
+                      <i style="margin-right: 5px;" class="fa fa-times"></i>Delete
+                    </button>
                   </div>
                   <div class="col-md-1">
+                    <!-- TODO(samchrisinger): pin down behavior here
```

Comment:
Is this TODO still relevant? I'm happy to be a sounding board if you need someone to discuss ideas with on this part.